src/main/java/org/prebid/server/bidder/rubicon/RubiconBidder.java

```diff
@@ -331,7 +331,8 @@ private Imp makeImp(Imp imp, ExtImpPrebid extPrebid, ExtImpRubicon extRubicon,
         if (isVideo(imp)) {
             builder
                     .banner(null)
-                    .video(makeVideo(imp.getVideo(), extRubicon.getVideo(), extPrebid));
+                    .video(makeVideo(imp.getVideo(), extRubicon.getVideo(), extPrebid,
```

Comment:
Please consider passing whole `imp` and `site` to `makeVideo` and let this method pull out everything it needs from these objects.